packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -0,0 +1,65 @@
+|                           | --pf-v5-global--palette--black-800 | #1b1d21 |
+|                           | --pf-v5-global--palette--black-850 | #212427 |
+|                           | --pf-v5-global--palette--black-900 | #0f1214 |
+|                           | --pf-v5-global--palette--black-1000| #030303 |
+|New red colors added       | --pf-v5-global--palette--red-9999  | #fe5142 |
+|                           | --pf-v5-global--palette--red-8888  | #ff7468 |
+|New primary blue added     | --pf-v5-global--palette--blue-300 | #1fa7f8 |
+
+
+
+**Global variables added**
+
+`--pf-v5-global--BackgroundColor--300` added as background color for elements that appear on top of others. Examples include popovers and dropdown menus.
+
+`--pf-v5-global--BackgroundColor--400` added as background color for hover/interaction state of items placed on top of other dark background. Examples include hover/focus backgrounds for items in a dropdown menu or popover and control elements like text inputs, textareas, menu toggles.
+
+`--pf-v5-global--BorderColor--400` added as a bottom border color used for control elements.
+
+**Status colors added**
+
+`--pf-v5-global--primary-color--300` added as a new primary background color that meets contrast requirements for elements that will have text on top, including primary buttons and unread badges.
+
+Here is a link to [all global var overrides](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/scss-variables.scss) for dark theme for your reference. 
+
+## Best practices
+
+The following are some best practices for designing and implementing applications to make it easier to support light and dark theming. We recommend that you keep these considerations in mind for any new development.
+
+* Using PatternFly components without customizations will make it easier to implement dark theme in your application. Most of the problems encountered have been around the use of custom styles, overrides to PatternFly styles, and non-PatternFly components. Whenever possible, to ensure consistent styling and behavior when switching themes, try and use PatternFly components as they are.
+
+
+* When making overrides, utilize the global and component CSS variable system to define the override. For example, to override a primary button’s background color, declare `.pf-v5-c-button { --pf-v5-c-button--m-primary--BackgroundColor: [color]; }` instead of `.pf-v5-c-button.pf-m-primary { background-color: [color]; }`.
+
+
+* When creating overrides and custom styles, try to utilize the CSS variable system for the style values. For example, use background-color: `var(--pf-v5-global--BackgroundColor--100)` for a primary background instead of `background-color: #fff` or `background-color: white`. Colors like `#fff` and `white` will not change between light and dark theme, but the use of `--pf-v5-global--BackgroundColor--100` will. 
```

Comment:
```suggestion
* When creating overrides and custom styles, try to utilize the CSS variable system for the style values. For example, use `background-color: var(--pf-v5-global--BackgroundColor--100)` for a primary background instead of `background-color: #fff` or `background-color: white`. Colors like `#fff` and `white` will not change between light and dark theme, but the use of `--pf-v5-global--BackgroundColor--100` will. 
```